editor/cypress/e2e/new-recipe.cy.ts

```diff
@@ -33,6 +33,62 @@ describe("New Recipe View", () => {
         cy.checkNamesInOrder([newRecipeTitle]);
       });
 
+      it("should be able to paste ingredients", () => {
+        cy.findByRole("heading", { name: "New Recipe" });
+
+        const newRecipeTitle = "My New Recipe";
+
+        cy.findAllByLabelText("Name").first().clear();
+        cy.findAllByLabelText("Name").first().type(newRecipeTitle);
+
+        cy.findByText("Paste Ingredients").click();
+        cy.findByTitle("Ingredients Paste Area").type(
+          `
+1 cup water ((for the dashi packet))
+1  dashi packet
+2 tsp sugar
+2 Tbsp mirin
+2 Tbsp soy sauce
+½  onion ((4 oz 113 g))
+1  green onion/scallion ((for garnish))
+3  large eggs (50 g each w/o shell)
+2  tonkatsu
+2 servings cooked Japanese short-grain rice ((typically 1⅔ cups (250 g) per donburi serving))
```

Comment:
Grabbed from the html fixture, which is from Just One Cookbook. Weird whitespace left in to confirm behavior and serve as a start for testing other features like automatically removing excess spaces and handling list bullets.